client-participation/Dockerfile

```diff
@@ -1,19 +1,24 @@
-FROM node:10.15.0
+# The version of node-sass in use requires node 8.x
+FROM node:8-alpine
```

Comment:
What's best practice? I would have thought patch versions, and use dependant to update them. Why not lock the versions more?

Reply:
There are very few changes in node patch updates other than security fixes, bug fixes, and performance improvements. Node 8 is not even active, but being on the latest security patch is the best practice. node:8-alpine implies the latest patch for 8.x.

Reply:
TIL Node 8 end-of-life was Dec 31 2019. So there is likely not going to be anything released beyond 8.17.0. I will update to just say 8.17.0

Reply:
I put patch-level node versions in all the Dockerfiles now

Reply:
Ah didnt realize that tag format was just leaving patch level relaxed..! That gets +1 from me :)